.github/workflows/main.yaml

```diff
@@ -29,8 +29,19 @@ jobs:
             echo "Defaulting to todays date ${releaseDate}"
           fi
 
+          echo "releaseDate=${releaseDate}" >> $GITHUB_ENV
```

Comment:
nit: as a reminder, does the cron job provide a value for the `releaseDate` input?